internal/command/createreleaseartifacts.go

```diff
@@ -38,7 +38,7 @@ type LibraryRelease struct {
 
 var CmdCreateReleaseArtifacts = &Command{
 	Name:  "create-release-artifacts",
-	Short: "Create release artifacts from a merged release PR",
+	Short: "Generate release artifacts from a merged PR.",
```

Comment:
Generate sounds like it's running a generator. This is about building/packaging...